go/src/socialapi/Makefile

```diff
@@ -227,6 +227,12 @@ testwebhook:
 	@./tests.test $(DBG) $(EXTRAS) -c $(CONFIG)
 	@rm ./tests.test
 
+testpresence:
+	@echo "$(OK_COLOR)--> presence tests... $(NO_COLOR)"
+	@`which go` test -race -c workers/presence/*.go
```

Comment:
I mostly wanted to prefer package paths over file paths:

socialapi/workers/presence > workers/presence/*.go